Sileo/Sileo/fr.lproj/Localizable.strings

```diff
@@ -96,46 +96,65 @@
 
 "Installed_Package_Header" = "Paquet installé";
 "Version" = "Version";
 "Show_Package_Contents_Button" = "Dévoiler les contenus du paquet";
 
+"Licenses_Page_Title" = "Licenses";
+
 "Settings_Payment_Provider_Heading" = "Fournisseurs de paiement";
 "Settings_Translations_Heading" = "Traductions";
 
 "Settings_Payment_Provider.Purchases_Heading" = "Achats";
 "Settings_Payment_Provider.No_Purchases" = "Aucun achat";
 
+"Settings" = "Paramètres";
+"Settings_Page_Title" = "Paramètres";
+"Dark_Mode_Toggle" = "Mode Sombre";
+"Theme" = "Thème";
+"Tint_Color" = "Couleur de la teinte";
+"Reset_Tint_Color" = "Restaurer la couleur de la teinte";
+"Auto_Complete_Queue" = "File d'attente à saisie automatique";
+"Enable_Analytics" = "Activer les données analytiques";
+
 "Cydia_Sign_In" = "Se connecter au Cydia Store.";
 "Payment_Provider_Signed_In" = "Connecté";
 "Payment_Provider_Signed_In_With_Name" = "Utilisateur connecté: %@"; // %@ = user's name
 "Payment_Provider_Sign_Out" = "Déconnexion";
 "Payment_Provider_Sign_Out_Confirm.Title" = "Déconnecter de %@"; // %@ = payment provider name
-"Payment_Provider_Sign_Out_Confirm.Body" = "Vous ne serez pas capable de télécharger vos paquets acheter avant de re-connecter.";
+"Payment_Provider_Sign_Out_Confirm.Body" = "Vous ne serez pas capable de récupérer vos paquets achetés avant de vous re-connecter.";
 
+"Show_Install_Details" = "Afficher les détails";
 "Show_Install_Details" = "Dévoiler les détails";
 "Hide_Install_Details" = "Cacher les détails";
 
-"After_Install_Relaunch" = "Relancer le Sileo";
+"After_Install_Relaunch" = "Relancer Sileo";
 "After_Install_Respring" = "Relancer le Springboard";
-"After_Install_Reboot" = "Recommencer l'appareil";
+"After_Install_Reboot" = "Redémarrer l'appareil";
 
 "Dangerous_Repo.Title" = "Source dangereuse";
-"Dangerous_Repo.Body" = "Vous essayez d'installer une source qui est connue être dangereuse pour des raisons qui peuvent inclure la piratie, l'hébergement de contenu illégale ou des virus.\n\nLes sources indiqués être dangereuses distribuent souvent des paquets ayant des effets inattendues, soit en forme de paquets payés rendus gratuits, ou d'autres paquets malicieux. Ceux-ci peuvent avoir des effects permanents sur votre appareil, et peuvent même réduire votre capacité pour installer un jailbreak. Nous recommendons que vous agissez avec prudence pour éviter des logiciels dangereux.";
+"Dangerous_Repo.Body" = "Vous essayez d'installer une source qui est connue pour être dangereuse (hébergeant des paquets piratés, du contenu illégal ou des virus).\n\nLes sources indiqués être dangereuses distribuent souvent des paquets ayant des effets inattendues, soit sous forme de paquets payant rendus gratuits, ou d'autres paquets malicieux. Ceux-ci peuvent avoir des effects permanents sur votre appareil et votre vie privée, et peuvent même altérer le fonctionnement d'un jailbreak. Nous vous recommandons d'agir avec prudence afin d'éviter des logiciels dangereux.";
```

Comment:
I think "indiqué**e**s **comme** dangereuses" would be better than "indiqués être dangereuses"